plugin/restore/mysql/mysql.go

```diff
@@ -188,66 +213,64 @@ func getBinlogReplayList(startBinlogInfo api.BinlogInfo, binlogDir string) ([]st
 		return nil, fmt.Errorf("cannot read binlog directory %s, error %w", binlogDir, err)
 	}
 
-	var toBeReplayedBinlogNames []string
-
+	var binlogFilesToReplay []BinlogFile
```

Comment:
What do you mean by the starting sequence?
Currently the "entire startBinlogInfo" is just the filename and position of the starting binlog file.

Reply:
I meant the line 206 ~ 209 may be better put in the caller, thus we are passing the seq to make getBinlogReplayList more readable. Because we are looking for a list of binlog files after a seq.

But with the new "binlogFilesToReplaySorted[0].Seq != startBinlogSeq" block, passing the entire info makes more sense now.